go.mod

```diff
@@ -3,9 +3,11 @@ module github.com/vimeo/caps
 go 1.12
 
 require (
+	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
```

Comment:
What's up with this, I don't see it used anywhere and I don't see any new imports?

Reply:
Looks like this already was [used here](https://github.com/vimeo/caps/blob/master/sami/parser_test.go#L9), but wasn't added properly in `go.mod` and got added when I ran `go mod tidy`.